services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowValidationNegativeSpec.groovy

```diff
@@ -161,4 +166,23 @@ class FlowValidationNegativeSpec extends BaseSpecification {
         def switches = topology.getActiveSwitches()
         return [switches.first(), switches.first()]
     }
+
+    @Unroll
+    def "Unable to #action a non-existent flow"() {
+        Assume.assumeFalse("Ignored due to issue #2027", action == "validate")
+        when: "Trying to #action rules with non-existent flow id"
```

Comment:
I still don't like the description, sorry :)
Can we get rid of 'rules' word here?